ChangeLog.d/8664.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * mbedtls_md_psa_alg_from_type() and mbedtls_md_type_from_psa_alg() helper
+     functions were added to convert from mbedtls_md_type_t to psa_algorithm_t
+     and viceversa.
```

Comment:
I _think_ that's two words in English: vice versa. https://dictionary.cambridge.org/dictionary/english/vice-versa

Reply:
Since this description was basically the same as the one I initially added into #8664, I will apply the same changes Gilles asked for that one ([here](https://github.com/Mbed-TLS/mbedtls/pull/8664#discussion_r1440847900) and [here](https://github.com/Mbed-TLS/mbedtls/pull/8664#discussion_r1440848427)).